core/templates/dev/head/components/side_navigation_bar/side_navigation_bar_directive.html

```diff
@@ -77,6 +77,9 @@
       height: 56px;
       padding-top: 6px;
     }
+    .oppia-sidebar-menu-icon.fa {
```

Comment:
Firstly, please follow the style of the sidebar classes above this one, in order to make sure the CSS is scoped only to this directive.

Secondly, it looks like the "fa" class affects only one of the sidebar elements. Why is this the case?